<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuitarScales</class>
 <widget class="QMainWindow" name="GuitarScales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuitarScales</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,10,1">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QComboBox" name="cbMode"/>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="cbNote">
        <property name="currentText">
         <string>A</string>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G#</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cbScaleDegrees">
        <property name="text">
         <string>Show Intervals</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/UnkindledTwo&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Github&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lScaleName">
      <property name="text">
       <string>C Major</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_Colors"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSet_Colors">
   <property name="text">
    <string>Set Colors</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
